### PR TITLE
redirect to overview after signing up

### PR DESCRIPTION
**Pull request by fzscodes**
# Description
Redirect to overview after signing up #209.

# How to test?
Sign up as a new user. On account creation alert is shown and user is redirected to overview page.

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by fzscodes** [comment]
second commit is about the latest from develop...

File: src/client/components/UserCreationSuccess/UserCreationSuccess.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 import React from 'react';
+import { Redirect } from 'react-router-dom';
 
 export default function UserCreationSuccess() {
-  return (
-    <div>
-      Your account was successfully created, you can now head over and sign in
-    </div>
-  );
+  alert('Your account was successfully created');
```
Comment (tpetrina) [inline src/client/components/UserCreationSuccess/UserCreationSuccess.js:6]: Ouch, try finding a package that does `toasts` for good practice.

**Comment by benna100** [comment]
Hey Fauzia...

Are you on this feedback?

**Comment by fzscodes** [comment]
> Hey Fauzia...
> 
> Are you on this feedback?

Hi Benjamin,
I am working on it and will implement the feedback by today evening

**Review by benna100 — APPROVED** [review]

**Comment by benna100** [comment]
There is a merge conflict here

**Review by benna100 — APPROVED** [review]

**Pull request merged**